lightning/backend/sql2kv_test.go

```diff
@@ -119,6 +119,36 @@ func (s *kvSuite) TestEncode(c *C) {
 	}))
 }
 
+func (s *kvSuite) TestEncodeTimestamp(c *C) {
+	ty := *types.NewFieldType(mysql.TypeDatetime)
+	ty.Flag |= mysql.NotNullFlag
+	c1 := &model.ColumnInfo{
+		ID:           1,
+		Name:         model.NewCIStr("c1"),
+		State:        model.StatePublic,
+		Offset:       0,
+		FieldType:    ty,
+		DefaultValue: "CURRENT_TIMESTAMP",
+		Version:      1,
+	}
+	cols := []*model.ColumnInfo{c1}
+	tblInfo := &model.TableInfo{ID: 1, Columns: cols, PKIsHandle: false, State: model.StatePublic}
+	tbl, err := tables.TableFromMeta(NewPanickingAllocator(0), tblInfo)
+	c.Assert(err, IsNil)
+
+	logger := log.Logger{Logger: zap.NewNop()}
+
+	encoder := NewTableKVEncoder(tbl, mysql.ModeStrictAllTables, 1234567893)
+	pairs, err := encoder.Encode(logger, nil, 70, []int{-1, 1})
+	c.Assert(err, IsNil)
+	c.Assert(pairs, DeepEquals, kvPairs([]kvenc.KvPair{
+		{
+			Key: []uint8{0x74, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x5f, 0x72, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x46},
+			Val: []uint8{0x8, 0x2, 0x9, 0x80, 0x80, 0x80, 0xf0, 0xfd, 0x8e, 0xf7, 0xc0, 0x19},
```

Comment:
How did this value derived 🤔

Reply:
Through TiDB's `EncodeValue` function (though in reality I just get the actual output and paste it back, as the purpose of this test is to ensure we get the same fixed value everytime).

Reply:
Ah, fixed value is a good test point.
If this reproduce step is short we could paste those steps here and let code generates this value